packages/flutter/lib/src/painting/matrix_utils.dart

```diff
@@ -22,20 +29,20 @@ class MatrixUtils {
     assert(transform != null);
     final Float64List values = transform.storage;
     // Values are stored in column-major order.
-    if (values[0] == 1.0 && // col 1
-        values[1] == 0.0 &&
-        values[2] == 0.0 &&
-        values[3] == 0.0 &&
-        values[4] == 0.0 && // col 2
-        values[5] == 1.0 &&
-        values[6] == 0.0 &&
-        values[7] == 0.0 &&
-        values[8] == 0.0 && // col 3
-        values[9] == 0.0 &&
-        values[10] == 1.0 &&
-        values[11] == 0.0 &&
-        values[14] == 0.0 && // bottom of col 4 (values 12 and 13 are the x and y offsets)
-        values[15] == 1.0) {
+    if (_nearEqual(values[0], 1.0) && // col 1
```

Comment:
did you mean to leave these? I thought from our discussion that getAsTranslation and friends were going to be moved back to the faster and more precise old code, and only developer debugging aids were going to use the rounding code.

Reply:
I just haven't had a chance to update this CL yet. I intend to keep these methods as is and fork off debugging aid only versions.